app/components/general/components/warning-callout/settings.json

```diff
@@ -0,0 +1,8 @@
+{
+    "componentName": "warningCallout",
+    "params": {
+        "heading": "Important",
+        "HTML": "Testing html"
+    }
+}
+  
```

Comment:
Not sure what's going on here but GutHub isn't happy – is there whitespace on the final line?

Reply:
ah there was a space on that one